Added Sherlock #283

sumanismcse opened the pull request:
Added Sherlock

sumanismcse committed:
Added Sherlock

sumanismcse committed:
Added Sherlock

Added Itemlink

brittanyrw commented:
Hi @sumanismcse, this looks good except your object should be placed in alphabetical order within the file. Right now you have it at the bottom of the file. Can you please move it up so that it is alphabetical? Then we can go ahead and get this merged! 🙂

sumanismcse committed:
Put Sherlock in alphabetical order

Actually I already fixed in my other pull request when i added matilda the musical but it has not been merged .

sumanismcse commented:
Actually, I already fixed in my other pull request when i added matilda the musical but that pull request has not been merged .
